airbyte-integrations/connectors/source-facebook-marketing/integration_tests/spec.json

```diff
@@ -352,4 +352,4 @@
       "oauthFlowOutputParameters": [["access_token"]]
     }
   }
-}
+}
```

Comment:
please exclude this file from your changes

Reply:
Seconding this, please remove the file from this PR.